src/components/Nodes/GethConfig/VersionList.jsx

```diff
@@ -105,11 +105,20 @@ class VersionList extends Component {
     const { client } = this.props
     const { release } = client
     if (selectedRelease) {
+      // Set selectedRelease passed into func
       this.setSelectedRelease(selectedRelease)
-      // Remove selectedRelease from remoteReleases so there are no duplicates in the list
+      // Remove selectedRelease from remoteReleases,
+      // so there are no duplicates in the list
       this.dedupedRemoteReleases()
```

Comment:
nit: function name easier to reason about as a verb than a noun. not part of this change set, though.